src/server/Server.ts

```diff
@@ -1,31 +1,29 @@
-const { loggerFactory } = require('./logger/log4js');
-const { setupExpressApp } = require('./ExpressApp');
-const MailHandler = require('./handler/MailHandler');
-const asHttpsServer = require('./ssl/index');
+import { loggerFactory } from './logger/log4js';
+import { setupExpressApp } from './ExpressApp';
+import MailHandler from './handler/MailHandler';
+import asHttpsServer from './ssl/index';
 
 const logger = loggerFactory('Server');
 
-const authenticationUsers = (usersString) => {
+const authenticationUsers = (usersString: string): { [x: string]: string; } => {
 
   const authenticationParams = usersString.split(';');
 
-  const users = authenticationParams.flatMap(auth => {
-
+  const users = authenticationParams.map(auth => {
     const userPasswordPair = auth.split(':');
-
-    return {
-      [userPasswordPair[0]]: userPasswordPair[1]
-    };
+    return { [userPasswordPair[0]]: userPasswordPair[1] };
   });
 
-  return Object.assign(...users);
+  return users.reduce((acc, curr) => {
+    return { ...acc, ...curr };
+  }, {});
 };
 
 const mailHandler = new MailHandler(process.env.MAIL_HISTORY_DURATION);
 
 const apiAuthentication = process.env.AUTHENTICATION
   ? { enabled: true, users: authenticationUsers(process.env.AUTHENTICATION) }
-  : { enabled: false };
+  : { enabled: false, users: {} };
```

Comment:
Couldn't that remain undefined, resolved internally then?

```suggestion
  : { enabled: false };
```